pom.xml

```diff
@@ -218,6 +220,25 @@
             <version>${shrinkwrap.version}</version>
         </dependency>
 
+        <!-- SCM utils -->
+        <dependency>
```

Comment:
any additional advantage we get using mvn scm providers compared to use using respective providers ?